mongorestore/restore.go

```diff
@@ -157,7 +301,7 @@ func (restore *MongoRestore) RestoreIntent(intent *intents.Intent) Result {
 			if strings.HasPrefix(intent.C, "system.") {
 				log.Logvf(log.Always, "cannot drop system collection %v, skipping", intent.Namespace())
 			} else {
-				log.Logvf(log.Info, "dropping collection %v before restoring", intent.Namespace())
+				log.Logvf(log.Always, "dropping collection %v before restoring", intent.Namespace())
```

Comment:
any particular reason this was bumped up to `log.Always`?

Reply:
Yeah, I thought it made sense for this message in particular to always be logged. I think it gives good information about what restore is doing, and it's good to be explicit when taking a destructive action.

There's a larger question here about how we make the choice about what verbosity level messages should use.